bindings/netstandard/ElectionGuard/ElectionGuard.Decryption.Tests/Decryption/DecryptionMediatorExtensions.cs

```diff
@@ -12,6 +12,31 @@ public static class DecryptionMediatorExtensions
     /// Run the decryption process
     /// </summary>
     public static DecryptionResult RunDecryptionProcess(
+    this DecryptionMediator mediator,
+        TestDecryptionData data,
+        List<Guardian> quorumOfGuardians,
+        bool tallyOnly = false)
+    {
+        return tallyOnly
+            ? mediator.RunDecryptionTallyOnly(data, quorumOfGuardians)
+            : mediator.RunDecryptionProcessWithBallots(data, quorumOfGuardians);
+    }
+
+    public static DecryptionResult RunDecryptionTallyOnly(
+    this DecryptionMediator mediator, TestDecryptionData data, List<Guardian> quorumOfGuardians)
+    {
+        // Create Shares
+        foreach (var guardian in quorumOfGuardians)
+        {
+            var shares = guardian.ComputeDecryptionShares(
```

Comment:
added the override that returns the same object but the ballot shares array will be empty

Reply:
changed my mind